docs: tw798 (#3270)
OffchainLabs/arbitrum-docs: docs/partials/glossary/_arbified-token-list.mdx

titleforSort: Arbified Token List
---

> A token list that conforms to [Uniswap's token list specification](https://github.com/Uniswap/token-lists); Arbified lists are generated by inputting externally maintained list ([i.e., coinmarketcap's list](https://api.coinmarketcap.com/data-api/v3/uniswap/all.json)) and outputting a list that includes all of the instances of token contracts on the Arbitrum chain bridged via the canonical [Arbitrum Token Bridge](/intro/glossary#arb-token-bridge) from tokens on the input list. (See code [here](https://github.com/OffchainLabs/arbitrum-token-lists).)

Reviewer: ```suggestion
A token list that conforms to [Uniswap's token list specification](https://github.com/Uniswap/token-lists); Arbified lists are generated by inputting an externally maintained list ([that is, CoinMarketCap's list](https://api.coinmarketcap.com/data-api/v3/uniswap/all.json)) and outputting a list that includes all of the instances of token contracts on the Arbitrum chain bridged via the canonical [Arbitrum Token Bridge](/intro/glossary#arb-token-bridge) from tokens on the input list. (See the [arbitrum-token-lists source code](https://github.com/OffchainLabs/arbitrum-token-lists).)
```
